src/pages/snippets/deferred-loading-in-angular.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: What is Deferred Loading in Angular
+description: "Deferred Loading in Angular"
+tags: ["angular", "Deferred Loading", "Zoneless angular"]
+pubDate: Jul 22, 2023
+contributedBy: "@pawanpatil08"
+---
+
+as a part of our ongoing work on implementing the Signals RFC,
```

Comment:
I see a lot of minor grammar issues / non-capital letters. Please run this through Grammarly or Google Docs since your actual content is well-written but has many obvious issues which can be fixed automatically.

Reply:
Hey, @ajitzero  tried it in Google Docs.
please suggest the any mistake in this file

Reply:
Updated.